Cannot read property 'references' of undefined

I've got some issues when running this action on my repo :(

```
##[error]TypeError: Cannot read property 'references' of undefined
##[error]Node run failed with exit code 1
```

Here is my `yml` file : 
```
name: generate plantuml
on: [push]

jobs:
  generate_plantuml:
    runs-on: ubuntu-latest
    name: plantuml
    steps:
    - name: checkout
      uses: actions/checkout@v1
      with:
        fetch-depth: 1
    - name: plantuml
      id: plantuml
      uses: grassedge/generate-plantuml-action@v1.1
      env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

--- a/src/utils.ts
+++ b/src/utils.ts
@@ -26,7 +26,7 @@ const infoRegexp = /^plantuml(?:@(.+))?:([\w-_.]+)/;
 
 function puFromMd(markdown) {
     const md = new markdownit();
-    const fences = md.parse(markdown)
+    const fences = md.parse(markdown, {})
         .filter(token => token.type === 'fence')
         .filter(token => infoRegexp.test(token.info));
 

